### PR TITLE
renovate json

### PR DESCRIPTION
**Pull request by r-hannuschka**
update renovate.json to make renovate make merge requests into development

### CODE REVIEW

**Comment by konne** [comment]
@thomashaenig  please review

File: renovate.json [hunk 1]
```diff
@@ -9,5 +9,8 @@
         "packagePatterns": ["tslint", "prettier"],
         "groupName": "linters"
       }
+    ],
+    "baseBranches": [
+      "devlopment"
```
Comment (rarkins) [inline renovate.json]: This is a typo - should be "development"

**Comment by rarkins** [comment]
@r-hannuschka @konne please note there is one other Renovate configuration problem you might want to address. Right now it's configured for branch automerge however you have also enforced mandatory PR reviews on the repository, and this blocks automerging with the following error:

```
    "body": {
        "message": "Required status check \"Codacy/PR Quality Review\" is expected. At least 2 approving reviews are required by reviewers with write access.",
        "documentation_url": "https://help.github.com/articles/about-protected-branches"
    },
```

What then happens is that Renovate raises it as a PR instead, with a warning note as you can see here: https://github.com/q2g/q2g-ext-bookmark/pull/49

My suggestion: remove automerging from config, or keep automerge but leave it as the default PR-based automerge so that it can once the PR gets the required status checks/reviews.

**Comment by konne** [comment]
@rarkins thank you very much for the input @r-hannuschka  will fix it very soon

**Pull request merged**